### ISSUE TITLE
Coverage is always less than ratio with any minimum-ratio value

### ISSUE THREAD

**Issue by superical** [barecheck/code-coverage-action#54]
Congratulations on a very cool action! I think this will be very useful for many people.

I have found that the action always fail with `Code coverage is less than minimum code coverage ratio` whenever I set `minimum-ratio` to anything more than zero. It fails even when I set `minimum-ratio` to a ridiculously high number.

Here's a snippet of the output:

```
Compare branch code coverage: 70.50%
Base branch code coverage: 80.51%
Code coverage diff: -10.01%
send-summary-comment is enabled for this workflow
minimum-ratio: 10000
minimum-ratio is enabled for this workflow
Error: Code coverage is less than minimum code coverage ratio
```

**Comment by superical** [comment]
After some investigation, it seems like there is an issue with type of `coverageDiff` in `checkMinimumRatio` being a string instead of a number. The resultant value of `coverageDiffAlert` becomes a concatenation when what we really want is an addition.

Here's an output from what I have tested so far:

```
Compare branch code coverage: 70.50%
Base branch code coverage: 80.51%
Code coverage diff: -10.01%
send-summary-comment is enabled for this workflow
minimum-ratio: 10000
minimum-ratio is enabled for this workflow
typeof coverageDiff: string
typeof minCoverageRatio: number
coverageDiffAlert: -10.0110000
Error: Code coverage is less than minimum code coverage ratio
```

Notice that `coverageDiff` is a string. This is how the action always fail at any `minimum-ratio` more than zero because `coverageDiffAlert` will always be negative. I have tested converting `coverageDiff` into a number and it works. I might create a PR to demonstrate what I mean when I have time.
